<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>

<head>
  <meta name=viewport content="width=800">
  <meta name="generator" content="HTML Tidy for Linux/x86 (vers 11 February 2007), see www.w3.org">
  <style type="text/css">
    /* Color scheme stolen from Sergey Karayev */
    
    a {
      color: #1772d0;
      text-decoration: none;
    }
    
    a:focus,
    a:hover {
      color: #f09228;
      text-decoration: none;
    }
    
    body,
    td,
    th,
    tr,
    p,
    a {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px
    }
    
    strong {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px;
    }
    
    heading {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 22px;
    }
    
    papertitle {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 14px;
      font-weight: 700
    }
    
    name {
      font-family: 'Lato', Verdana, Helvetica, sans-serif;
      font-size: 32px;
    }
    
    .one {
      width: 160px;
      height: 160px;
      position: relative;
    }
    
    .two {
      width: 160px;
      height: 160px;
      position: absolute;
      transition: opacity .2s ease-in-out;
      -moz-transition: opacity .2s ease-in-out;
      -webkit-transition: opacity .2s ease-in-out;
    }
    
    .fade {
      transition: opacity .2s ease-in-out;
      -moz-transition: opacity .2s ease-in-out;
      -webkit-transition: opacity .2s ease-in-out;
    }
    
    span.highlight {
      background-color: #ffffd0;
    }
  </style>
  <link rel="icon" type="image/png" href="images/UWM.png">
  <title>Mingren Shen</title>
  <meta http-equiv="Content-Type" content="text/html; charset=us-ascii">
  <link href='https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
</head>

<body>
  <!--         
=====================================================================
                       Personal Information
==================================================================== 
-->

  <table width="800" border="0" align="center" cellspacing="0" cellpadding="0">
    <tr>
      <td>
        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="67%" valign="middle">
              <p align="center">
                <name>Mingren Shen</name>
              </p>
              <p>I am a Ph.D. student in Materials Science  at <a href="https://www.wisc.edu/">UW-Madison</a>, where I work on applying computer vision and machine learning methods into material and related area problems.
              </p>
              <p>
                 Before UW-Madison, I got B.S. Physics at <a href="https://www.nju.edu.cn/EN/">Nanjing University</a> and M.S. Physics at <a href="http://english.ucas.ac.cn/">University of Chinese Academy of Sciences</a> where I was advised by <a href="http://sm08.iphy.ac.cn/member.html"> Dr. Ke Chen </a> and <a href="http://sm08.iphy.ac.cn/member.html"> Dr. Mingcheng Yang </a>.
              </p>

              <p>I obtained M.S. in Computer Sciences at <a href="https://www.wisc.edu/">UW-Madison</a> in May 2019. Cheers!</p>
              <p>

              <p>I obtained M.S. in Material Sciences and Engineering at <a href="https://www.wisc.edu/">UW-Madison</a> in December 2020. Cheers!</p>
              <p>
                
              </p>

<!--         
=====================================================================
                        Contact
==================================================================== 
-->
              <p align=center>
                <a href="mailto:mshen32@wisc.edu">Email</a> &nbsp/&nbsp
                <a href="https://github.com/iphyer/Mingren_Website/blob/master/data/cv_MingrenShen.pdf">CV</a> &nbsp/&nbsp
                <!-- <a href="data/JonBarron-bio.txt">Biography</a> &nbsp/&nbsp -->
                <a href="https://scholar.google.com/citations?user=_HeXbeQAAAAJ&hl=en">Google Scholar</a> &nbsp/&nbsp
                <a href="https://linkedin.com/in/mingren-shen"> LinkedIn </a>
              </p>
            </td>
            <!-- <td width="33%">
              <img src="images/JonBarron_circle.jpg">
            </td> -->
          </tr>
        </table>

<!--         
=====================================================================
                        Research
==================================================================== 
-->
        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td width="100%" valign="middle">
              <heading>Current Research</heading>
              <p>
                I'm interested in computer vision, machine learning, image processing, virtual reality, and computational material and biology. Much of my research is about applying newly developed computational methods and models for the physical world including DNA, TEM images etc.
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/ResearchProjects/CFD.png" alt="Medical" width="160" height="160"></td>
            <td width="75%" valign="top">

                <papertitle>Computing High-Fidelity PDE Solutions on Coarse Meshes using Machine Learning</papertitle>
                <br>
               <strong>Minren Shen</strong>, Summer 2021 Internship
               <br> Advisor: Rishikesh Ranade, Jay Pathak, Ansys Inc
                <p>
                <br> 
                Computational fluid dynamics simulations are important tools for studying of fluid flow involved phenomena, such as climate changes, weather systems, nuclear fusion plasma, vehicle aerodynamics structure designs etc. All those processes can be described by Naiver-Stokes Equation but to solve Naiver-Stokes Equation in fine scale requires huge amounts of computation resources. Here we propose a machine learning strategy to calculate high fidelity solutions on coarse meshes. Our approach uses a convolutional neural network to correct the lower resolution 3D flow fields computed by PDE solvers to higher resolution ones with adding of small-scale features that are not contained in lower resolution simulations. The correction is predicted on each mesh element and is a function of the solution fields and their gradients on that element as well as its neighbors.  Initial results indicate the neural network can accelerate the CFD simulations with fewer resources and generate flow field that is comparable to higher resolution simulations. This study shows that applying machine learning tools could pave the way for faster CFD simulations.
                 </br>
                </p>
            </td>
          </tr>
        </table>

<!-- 
=====================================================================
                        Publications
==================================================================== 
-->

        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td>
              <heading>Publications Lists and Abstract</heading>
            </td>
          </tr>
        </table>

<table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>

        <ol type="1" style="line-height:150%">

      <li>
          Awe, Adam M., Michael M. Vanden Heuvel, Tianyuan Yuan, Victoria R. Rendell, <strong>Mingren Shen</strong>, Agrima Kampani, Shanchao Liang, Dane D. Morgan, Emily R. Winslow, and Meghan G. Lubner. "Machine learning principles applied to CT radiomics to predict mucinous pancreatic cysts." Abdominal Radiology (2021): 1-11.
      </li>

      <li>
          <strong>Mingren Shen</strong>, Guanzhao Li, Dongxia Wu, Yuhan Liu, Jacob Greaves, Wei Hao, Nathaniel J. Krakauer, Leah Krudy, Jacob Perez, Varun Sreenivasan, Bryan Sanchez, Oigimer Torres, Wei Li, Kevin Field, and Dane Morgan. "Multi defect detection and analysis of electron microscopy images with deep learning." Computational Materials Science 199 (2021): 110576.
      </li>

      <li>
        	<strong>Mingren Shen</strong>, Guanzhao Li, Dongxia Wu, Yudai Yaguchi, Jack C. Haley, Kevin G. Field, and Dane Morgan. "A deep learning based automatic defect analysis framework for In-situ TEM ion irradiations." Computational Materials Science 197 (2021): 110560.
    	</li>

      <li>
          Kevin G. Field, Ryan Jacobs, <strong>Mingren Shen</strong>, Matthew Lynch, Priyam Patki, Christopher Field, and Dane Morgan. "Development and Deployment of Automated Machine Learning Detection in Electron Microcopy Experiments." Microscopy and Microanalysis 27, no. S1 (2021): 2136-2137.
      </li>

      <li>
        	Gurbani, Sidharth, Dane Morgan, Varun Jog, Leo Dreyfuss, <strong>Mingren Shen</strong>, Arighno Das, E. Jason Abel, and Meghan G. Lubner. "Evaluation of radiomics and machine learning in identification of aggressive tumor features in renal cell carcinoma (RCC)." Abdominal Radiology (2021): 1-11.

    	</li>

		  <li> 
  		  	Lawrence, Nick, <strong>Mingren Shen</strong>, Ruiqi Yin, Cloris Feng, and Dane Morgan. Exploring Generative Adversarial Networks for Image-to-Image Translation in STEM Simulation. arXiv preprint arXiv:2010.15315 (2020).
		  </li>

		  <li>
  		  	Xiaoyu Sun, Nathaniel J. Krakauer, Alexander Politowicz, Wei‐Ting Chen, Qiying Li, Zuoyi Li, Xianjia Shao, Alfred Sunaryo, <strong>Mingren Shen</strong>, James Wang, Dane Morgan. Assessing Graph‐based Deep Learning Models for Predicting Flash Point. Molecular Informatics (2020), 39 , 1900101.
      </li>

		  <li>
  		  	Liu, Yilin, Gregory R. Kirk, Brendon M. Nacewicz, Martin A. Styner, <strong>Mingren Shen</strong>, Dong Nie, Nagesh Adluru, Benjamin Yeske, Peter A. Ferrazzano, and Andrew L. Alexander. Harmonization and Targeted Feature Dropout for Generalized Segmentation: Application to Multi-site Traumatic Brain Injury Images. In Domain Adaptation and Representation Transfer and Medical Image Learning with Less Labels and Imperfect Data, pp. 81-89. Springer, Cham, 2019.
		  </li>

		  <li>
  		  	Luo, Guan-Zheng, Ziyang Hao, Liangzhi Luo, <strong>Mingren Shen</strong>, Daniela Sparvoli, Yuqing Zheng, Zijie Zhang et al. N 6-methyldeoxyadenosine directs nucleosome positioning in Tetrahymena DNA. Genome biology 19, no. 1 (2018): 200.
		  </li>

		  <li>
  		  	<strong>Mingren Shen</strong>, Fangfu Ye, Rui Liu, Ke Chen, Mingcheng Yang, and Marisol Ripoll. Chemically driven fluid transport in long microchannels. The Journal of chemical physics 145, no. 12 (2016): 124119
		  </li>


		  <li>
    		 	<strong>Mingren Shen</strong>, Liu Rui, Hou Mei-Ying, Yang Ming-Cheng, and Chen Ke. Mesoscale simulation of self-diffusiophoretic microrotor. ACTA PHYSICA SINICA 65, no.17 (2016).
		  </li>

		</ol>  
          </tr>
        </table>

          <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/pancreas.png" alt="ML-RCC" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://link.springer.com/article/10.1007/s00261-021-03289-0">
                <papertitle>Machine learning principles applied to CT radiomics to predict mucinous pancreatic cysts</papertitle>
                </a>
                <br>
                 Awe, Adam M., Michael M. Vanden Heuvel, Tianyuan Yuan, Victoria R. Rendell, <strong>Mingren Shen</strong>, Agrima Kampani, Shanchao Liang, Dane D. Morgan, Emily R. Winslow, and Meghan G. Lubner. 2021
                <p>
                  Current diagnostic and treatment modalities for pancreatic cysts (PCs) are invasive and are associated with patient morbidity. The purpose of this study is to develop and evaluate machine learning algorithms to delineate mucinous from non-mucinous PCs using non-invasive CT-based radiomics.
                <br>
                  <br> <a href="https://link.springer.com/article/10.1007/s00261-021-03289-0">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/fasterRCNN.gif" alt="ML-RCC" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://doi.org/10.1016/j.commatsci.2021.110576">
                <papertitle>Multi defect detection and analysis of electron microscopy images with deep learning</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong>, Guanzhao Li, Dongxia Wu, Yuhan Liu, Jacob Greaves, Wei Hao, Nathaniel J. Krakauer, Leah Krudy, Jacob Perez, Varun Sreenivasan, Bryan Sanchez, Oigimer Torres, Wei Li, Kevin Field, Dane Morgan, 2021
                <p>
                  Electron microscopy is widely used to explore defects in crystal structures, but human detecting of defects is often time-consuming, error-prone, and unreliable, and is not scalable to large numbers of images or real-time analysis. In this work, we discuss the application of machine learning approaches to find the location and geometry of different defect clusters in irradiated steels. We show that a deep learning based Faster R-CNN analysis system has a performance comparable to human analysis with relatively small training data sets. This study proves the promising ability to apply deep learning to assist the development of automated microscopy data analysis even when multiple features are present and paves the way for fast, scalable, and reliable analysis systems for massive amounts of modern electron microscopy data.
                <br>
                  <br> <a href="https://doi.org/10.1016/j.commatsci.2021.110576">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/YOLO.jpg" alt="ML-RCC" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://doi.org/10.1016/j.commatsci.2021.110560">
                <papertitle>A deep learning based automatic defect analysis framework for In-situ TEM ion irradiations</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong>, Guanzhao Li, Dongxia Wu, Yudai Yaguchi, Jack C.Haley, Kevin G.Field, Dane Morgan, 2021
                <p>

                	Videos captured using Transmission Electron Microscopy (TEM) can encode details regarding the morphological and temporal evolution of a material by taking snapshots of the microstructure sequentially. However, manual analysis of such video is tedious, error-prone, unreliable, and prohibitively time-consuming if one wishes to analyze a significant fraction of frames for even videos of modest length. In this work, we developed an automated TEM video analysis system for microstructural features based on the advanced object detection model called YOLO and tested the system on an in-situ ion irradiation TEM video of dislocation loops formed in a FeCrAl alloy. The system provides analysis of features observed in TEM including both static and dynamic properties using the YOLO-based defect detection module coupled to a geometry analysis module and a dynamic tracking module. Results show that the system can achieve human comparable performance with an F1 score of 0.89 for fast, consistent, and scalable frame-level defect analysis. This result is obtained on a real but exceptionally clean and stable data set and more challenging data sets may not achieve this performance. The dynamic tracking also enabled evaluation of individual defect evolution like per defect growth rate at a fidelity never before achieved using common human analysis methods. Our work shows that automatically detecting and tracking interesting microstructures and properties contained in TEM videos is viable and opens new doors for evaluating materials dynamics.

                <br>
                  <br> <a href="https://doi.org/10.1016/j.commatsci.2021.110560">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

                <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/syntheticFlow.png" alt="ML-RCC" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://www.cambridge.org/core/journals/microscopy-and-microanalysis/article/development-and-deployment-of-automated-machine-learning-detection-in-electron-microcopy-experiments/610B38A9A2C1F4942F04DC589B59217D/share/000e22dde85e77dd28d25ca2a67c576d66561558">
                <papertitle>Development and Deployment of Automated Machine Learning Detection in Electron Microcopy Experiments</papertitle>
                </a>
                <br>
                Kevin G. Field, Ryan Jacobs, <strong>Mingren Shen</strong>, Matthew Lynch, Priyam Patki, Christopher Field, Dane Morgan, 2021
                <p>
                Here, we demonstrate on-going pursuits toward providing automation in electron microscopy images using CNNs. The first pursuit is the application of a lightweight, reduced ordered transmission electron microscopy image simulation method to generate large amounts of "synthetic" data for training common CNN architectures. Specifically, we have developed a Python-based image generation algorithm that uses simplified phase-contrast models to simulate features of interest in either an under- or over-focused bright field Transmission Electron Microscopy (BF-TEM) condition. The output of this physics-based simulation is then integrated with experimentally obtained background images to create novel "synthetic" images with automatically labeled (via both bounding box and pixel segmentation) features. 
                <br>
                  <br> <a href="https://www.cambridge.org/core/journals/microscopy-and-microanalysis/article/development-and-deployment-of-automated-machine-learning-detection-in-electron-microcopy-experiments/610B38A9A2C1F4942F04DC589B59217D/share/000e22dde85e77dd28d25ca2a67c576d66561558">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/rcc.png" alt="ML-RCC" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://link.springer.com/article/10.1007/s00261-021-03083-y">
                <papertitle>Evaluation of radiomics and machine learning in identification of aggressive tumor features in renal cell carcinoma (RCC)</papertitle>
                </a>
                <br>
                Gurbani, Sidharth, Dane Morgan, Varun Jog, Leo Dreyfuss, <strong>Mingren Shen</strong>, Arighno Das, E. Jason Abel, and Meghan G. Lubner, 2021
                <p>

                The purpose of this study was to evaluate the use of CT radiomics features and machine learning analysis to identify aggressive tumor features, including high nuclear grade (NG) and sarcomatoid (sarc) features, in large renal cell carcinomas (RCCs).

                The XG Boost Model performed best on the tested data. After manual and machine feature extraction, models consisted of 3, 7, 5, 10 radiomics features for NC sarc, PV sarc, NC NG and PV NG, respectively. The area under the receiver operating characteristic curve (AUC) for these models was 0.59, 0.65, 0.69 and 0.58 respectively. The multichannel NG model extracted 6 radiomic features using the feature selection strategy and showed an AUC of 0.67.

                Statistically significant but weak associations between aggressive tumor features (high nuclear grade, sarcomatoid features) in large RCC were identified using 3D radiomics and machine learning analysis.
                <br>
                  <br> <a href="https://link.springer.com/article/10.1007/s00261-021-03083-y">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/gan.gif" alt="STEM-GAN" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://arxiv.org/abs/2010.15315">
                  <papertitle>Exploring Generative Adversarial Networks for Image-to-Image Translation in STEM Simulation</papertitle>
                </a>
                <br>
                Nick Lawrence, <strong>Mingren Shen</strong>, Ruiqi Yin, Cloris Feng, Dane Morgan, 2020
                <p>
                <br> The use of accurate scanning transmission electron microscopy (STEM) image simulation methods require large computation times that can make their use infeasible for the simulation of many images. Other simulation methods based on linear imaging models, such as the convolution method, are much faster but are too inaccurate to be used in application. In this paper, we explore deep learning models that attempt to translate a STEM image produced by the convolution method to a prediction of the high accuracy multislice image. We then compare our results to those of regression methods. We find that using the deep learning model Generative Adversarial Network (GAN) provides us with the best results and performs at a similar accuracy level to previous regression models on the same dataset. Codes and data for this project can be found in this GitHub repository, 
                <a href="https://github.com/uw-cmg/GAN-STEM-Conv2MultiSlice"> this https URL </a>
                <br>
                  <br> <a href="https://arxiv.org/abs/2010.15315">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

          <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/GNNFlashPoint.jpg" alt="Turbine" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://onlinelibrary.wiley.com/doi/10.1002/minf.201900101">
                  <papertitle>Assessing Graph‐based Deep Learning Models for Predicting Flash Point</papertitle>
                </a>
                <br>
                Xiaoyu Sun,  Nathaniel J. Krakauer,  Alexander Politowicz,  Wei‐Ting Chen,  Qiying Li,  Zuoyi Li,  Xianjia Shao,  Alfred Sunaryo,  <strong>Mingren Shen</strong>,  James Wang,  Dane Morgan, 2020
                <p>
                <br> Flash points of organic molecules play an important role in preventing flammability hazards and large databases of measured values exist, although millions of compounds remain unmeasured. To rapidly extend existing data to new compounds many researchers have used quantitative structure‐property relationship (QSPR) analysis to effectively predict flash points. In recent years graph‐based deep learning (GBDL) has emerged as a powerful alternative method to traditional QSPR. In this paper, GBDL models were implemented in predicting flash point for the first time. We assessed the performance of two GBDL models, message‐passing neural network (MPNN) and graph convolutional neural network (GCNN), by comparing against 12 previous QSPR studies using more traditional methods. Our result shows that MPNN both outperforms GCNN and yields slightly worse but comparable performance with previous QSPR studies. The average R^2 and Mean Absolute Error (MAE) scores of MPNN are, respectively, 2.3 % lower and 2.0 K higher than previous comparable studies. To further explore GBDL models, we collected the largest flash point dataset to date, which contains 10575 unique molecules. The optimized MPNN gives a test data R^2 of 0.803 and MAE of 17.8 K on the complete dataset. We also extracted 5 datasets from our integrated dataset based on molecular types (acids, organometallics, organogermaniums, organosilicons, and organotins) and explore the quality of the model in these classes.
                <br>
                  <br> <a href="https://onlinelibrary.wiley.com/doi/10.1002/minf.201900101">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/tfd.png" alt="Turbine" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://link.springer.com/chapter/10.1007/978-3-030-33391-1_10">
                  <papertitle>Harmonization and Targeted Feature Dropout for Generalized Segmentation: Application to Multi-site Traumatic Brain Injury Images</papertitle>
                </a>
                <br>
                Yilin Liu, Gregory R. Kirk, Brendon M. Nacewicz, Martin A. Styner, <strong>Mingren Shen</strong>, Dong Nie, Nagesh Adluru, Benjamin Yeske, Peter A. Ferrazzano, Andrew L. Alexander, 2019
                <p>
                <br> While learning based methods have brought extremely promising results in medical imaging, a major bottleneck is the lack of generalizability. Medical images are often collected from multiple sites and/or protocols for increasing statistical power, while CNN trained on one site typically cannot be well-transferred to others. Further, expert-defined manual labels for medical images are typically rare, making training a dedicated CNN for each site unpractical, so it is important to make best use of the limited labeled source data. To address this problem, we harmonize the target data using adversarial learning, and propose targeted feature dropout (TFD) to enhance the robustness of the model to variations in target images. Specifically, TFD is guided by attention to stochastically remove some of the most discriminative features. Essentially, this technique combines the benefits of attention mechanism and dropout, while it does not increase parameters and computational costs, making it well-suited for small neuroimaging datasets. We evaluated our method on a challenging Traumatic Brain Injury (TBI) dataset collected from 13 sites, using labeled source data of only 14 healthy subjects. Experimental results confirmed the feasibility of using the Cycle-consistent adversarial network for harmonizing multi-site MR images, and demonstrated that TFD further improved the generalization of the vanilla segmentation model on TBI data, reaching comparable accuracy with that of the supervised learning. The code is available at <a href="https://github.com/YilinLiu97/Targeted-Feature-Dropout.git">
                  <papertitle>here</papertitle></a>
                <br>
                  <br> 
                  <a href="https://link.springer.com/chapter/10.1007/978-3-030-33391-1_10">
                  <papertitle>Paper Link</papertitle></a>
                  </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/turbine.jpg" alt="Turbine" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://doi.org/10.1103/PhysRevApplied.12.034051">
                  <papertitle>Diffusive-Flux-Driven Microturbines by Fore-and-Aft Asymmetric Phoresis</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong>, Rui Liu, Ke Chen, and Mingcheng Yang, 2019
                <p>
                <br> We designed the microscale turbine that is powered purely by an external diffusive heat or mass flux. The turbine rotates perpendicular to the external thermal or chemical gradient due to fore-and-aft asymmetric thermophoresis or diffusiophoresis. Thus, its performance does not need alignment. The prototype turbine is validated by means of computer simulations. The results show that the angular velocity of the turbine sensitively depends on its structure and interactions with the solvent. This perpendicular-axis phoretic turbine provides large flexibility and the possibility to exploit existing thermal or chemical energies at small scales.
                <br>
                  <br> <a href="https://doi.org/10.1103/PhysRevApplied.12.034051">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

          <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/DNA.gif" alt="DNA" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://doi.org/10.1186/s13059-018-1573-3">
                  <papertitle>N6-methyldeoxyadenosine directs nucleosome positioning in Tetrahymena DNA</papertitle>
                </a>
                <br>
                Guan-Zheng Luo, Ziyang Hao, Liangzhi Luo, <strong>Minren Shen</strong>, Daniela Sparvoli, Yuqing Zheng, Zijie Zhang, Xiaocheng Weng, Kai Chen, Qiang Cui, Aaron P. Turkewitz and Chuan He, 2018
                <p>
                <br> I use large scale GPU accelerated molecular dynamics (MD) simulations to study mechanical property changes due to N6-methyldeoxyadenosine of DNA chains. The results shows knockout of a potential 6mA methyltransferase leads to a transcriptome-wide change of gene expression.
                <br>
                  <br> <a href="https://doi.org/10.1186/s13059-018-1573-3">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/jcp.gif" alt="JCP2016" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://aip.scitation.org/doi/abs/10.1063/1.4963721">
                  <papertitle>Chemically driven fluid transport in long micro channels</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong>, Fangfu Ye, Rui Liu, Ke Chen, Mingcheng Yang, and Marisol Ripoll, 2016
                <p>
                  <br> I show that a concentration drop across micro channels with periodically inhomogeneous boundary walls can laterally transport fluids over arbitrarily long distances along the micro channel. This work thus presents new insight into the fluid transport in long micro channels commonly found in nature and is useful for designing novel micro- or nanofluidic pumps. 
                  <br>
                  <br> <a href="https://aip.scitation.org/doi/abs/10.1063/1.4963721">
                  <papertitle>Paper Link</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

          <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/Publications/rotor.gif" alt="CHN" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="http://wulixb.iphy.ac.cn/CN/abstract/abstract68321.shtml#">
                  <papertitle>Mesoscale simulation of self-diffusiophoretic micro rotor</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong>, Rui Liu, Mei-Ying Hou , Ming-Cheng Yang , Ke Chen, 2016
                <p>
                  <br> I employ hybrid molecular dynamics (MD) simulations and multi-particle collision dynamics (MPC) to investigate the motion of micro rotors. The rotational direction and speed of the micro rotor are determined by bead-solvent interactions, the rotor geometry, the solvent viscosity and the catalytic reaction ratio.
                  <br>
                  <br> <a href="http://wulixb.iphy.ac.cn/CN/abstract/abstract68321.shtml#">
                  <papertitle>Paper Link(In Chinese)</papertitle></a>
                 </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

<!--         
=====================================================================
                        Conference
==================================================================== 
-->

<table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
  <tr>
    <td>
      <heading>Conference</heading>
    </td>
  </tr>
</table>

  <table width="100%" align="center" border="0" cellpadding="20">

<!-- No5 -->
 <tr>
    <!-- <td width="25%"><img src="images/Publications/DNA.gif" alt="DNA" width="160" height="160"></td>-->
    <td width="75%" valign="top"> 
        <a href="https://www.tms.org/mines2021">
          <Meeting>MiNES2021: Discerning the effects of solute additions in FeCrAl on dislocation dynamics under irradiation using a Machine Learning object detection algorithm</Meeting>
        </a>
        <p>
        American Nuclear Society (ANS), September 19–23, 2021 in Pittsburgh, Pennsylvania, USA
        </p>
        <p>
        Priyam Patki, <strong>Mingren Shen</strong>, Yudai Yaguchi, Jack Haley, Dane Morgan, Kevin Field
        </p>
        <p>Poster</p>
      </td>
  </tr>

<!-- No4 -->
 <tr>
    <!-- <td width="25%"><img src="images/Publications/DNA.gif" alt="DNA" width="160" height="160"></td>-->
    <td width="75%" valign="top"> 
        <a href="https://abdominalradiology.org/sar-subpages/annual-meeting/">
          <Meeting>SAR21: Use of radiomics and machine learning for identification of mucinous pancreas cysts</Meeting>
        </a>
        <p>
        Society of Abdominal Radiology, 2021, Virtual Meeting
        </p>
        <p>
        Adam Awe, Michael Vanden Heuvel, Tianyuan Yuan,  <strong>Mingren Shen</strong>, Victoria Rendell (MD), Agrima Kampani, Shanchao Liang, Dane Morgan (PhD), Emily Winslow (MD), and Meghan Lubner (MD)
        </p>
        <p>Oral Presentation</p>
      </td>
  </tr>

<!-- No3 -->
 <tr>
    <!-- <td width="25%"><img src="images/Publications/DNA.gif" alt="DNA" width="160" height="160"></td>-->
    <td width="75%" valign="top"> 
        <a href="https://www.arrs.org/am21">
          <Meeting>ARRS21: Diagnostic potential of machine learning analysis and radiomics for pancreatic cysts</Meeting>
        </a>
        <p>
        American Roentgen Ray Society, 2021, 
        </p>
        <p>
        Adam Awe, Michael Vanden Heuvel, Tianyuan Yuan, <strong>Mingren Shen</strong>, Victoria Rendell, Emily Winslow, Meghan Lubner
        </p>
        <p>Oral Presentation</p>
      </td>
  </tr>

<!-- No2 -->
  <tr>
    <!-- <td width="25%"><img src="images/Publications/DNA.gif" alt="DNA" width="160" height="160"></td>-->
    <td width="75%" valign="top"> 
        <a href="http://www.matscitech.org/">
          <Meeting>MS&T19: Automated Defect Detection in Electron Microscopy with Machine Learning</Meeting>
        </a>
        <p>
        Materials Science & Technology 2019 - Data Science for Material Property Interpretation (09/29/2019 - 10/03/2019, Portland, OR, USA)
        </p>
        <p>
        Dane Morgan, <strong>Mingren Shen</strong>, Wei Li, Kevin Field
        </p>
        <p>Oral Report</p>
      </td>
  </tr>

<!-- No1 -->
  <tr>
    <td width="75%" valign="top"> 
      <a href="https://cslide.ctimeetingtech.com/cirse2019/attendee/confcal/show/session/12237">
        <Meeting>CIRSE 2019: Identifying active extravasation on arteriograms using artificial intelligence</Meeting>
      </a>
      <p>
        Cardiovascular and Interventional Radiological Society of Europe (CIRSE) (09/07/2019 - 09/11/2019, Barcelona, Spain)
      </p>
      <p>
      <strong>Mingren Shen</strong>, Mesut Ozturk, Po-Ling Loh, Varun Jog, Paul Laeseke, Dane Morgan, Mark Kleedehn
      </p>
      <p>Oral Report: 2306.7 - Identifying active extravasation on arteriograms using artificial intelligence</p>
    </td>
  </tr>

</table>


<!--         
=====================================================================
                        Course Projects
==================================================================== 
-->
        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td>
              <heading>Course Projects</heading>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/CourseProjects/cs744.gif" alt="CS744" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://github.com/iphyer/FocousIngestOpt_FinalProject_CS744Fall2018">
                  <papertitle>Query time optimized video inference system</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong>, Shuoxuan Dong, Xiuyuan He, 2018
                <p>
                  <br> Optimizing the latency of a two CNNs video inference system by reusing the intermediate results of first CNN(ResNet50 ) to accelerate the calculation of second CNN( ResNet152 ). We successfully achieved 18% latency decrease without sacrificing the accuracy of the model.
                  <br> <a href="https://github.com/iphyer/FocousIngestOpt_FinalProject_CS744Fall2018/blob/master/FinalReport/Final_Report_744_Group_6.pdf">
                  <papertitle>Final Report</papertitle>
                </a>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/CourseProjects/cs760.gif" alt="CS760" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://github.com/iphyer/CS760_Twitter_Demographics">
                  <papertitle>Improvement of Twitter gender classifier using user mobility features</papertitle>
                </a>
                <br>
                Xinyi Liu , <strong>Mingren Shen</strong> , Faust Shi, 2017
                <p>
                  <br> We collected Twitter data from St Louis, MO to build a user gender classifier including Naïve Bayes Network, Random Forest, Ada Boost and Multi-Layer Neural Network (Python: scikit-learn) and test user mobility feature effects for those classifier by adding those geo features. </br>
                  <br> <a href="https://github.com/iphyer/CS760_Twitter_Demographics/blob/master/Report_final.pdf">
                  <papertitle>Final Report</papertitle></br></a>
                </p>
              </p>
            </td>
          </tr>
        </table>
<!--         
=====================================================================
                        Personal Projects
==================================================================== 
-->
        <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20">
          <tr>
            <td>
              <heading>Personal Projects</heading>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/PersonalProjects/BBQ.gif" alt="BBQ" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://github.com/iphyer/BBQ_Madhacks2018">
                  <papertitle>BBQ:Bounding Box Quality Checker</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong>, Xiuyuan He, 2018
                <p>
                  <br> 
                  BBQ standing bounding box qualifier which is a web service that examines the matching between ground truth bounding boxes and the predicted bounding boxes and produces a prediction quality report and debugging tools for object detection algorithm. A lot of work of object detection methods are focusing on how to generate bounding box annotations. However, few have talked about how to debug or check where the object detection model fails. This Hackathon project is a tool that can help people actually see where the object detection algorithm fails.
                  <br> 
                  <a href="https://github.com/iphyer/BBQ_Madhacks2018">
                  <papertitle>Github</papertitle>
                </a>
                </br>
                </p>
              </p>
            </td>
          </tr>
        </table>

        <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/PersonalProjects/UTD.gif" alt="UTD" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://github.com/iphyer/Drive-test-schedule-system">
                  <papertitle>Driver Test Schedule System</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong> , Yang Yang, Jiachen Shen, 2016
                <p>
                  <br> This project helps reminder the users when there are personalized available space for their driving test in College Town of TAMU. The project is build on Ruby on Rails,Ruby(4.2.2),Python. 
                  </br>
                  <br> <a href="https://github.com/iphyer/Drive-test-schedule-system">
                  <papertitle>Github</papertitle></br></a>
                </p>
              </p>
            </td>
          </tr>
        </table>

<!--           <table width="100%" align="center" border="0" cellpadding="20">
          <tr>
            <td width="25%"><img src="images/PersonalProjects/3k.gif" alt="3k" width="160" height="160"></td>
            <td width="75%" valign="top">
              <p>
                <a href="https://github.com/iphyer/Crack3kGREWords">
                  <papertitle>Crack GRE Words</papertitle>
                </a>
                <br>
                <strong>Mingren Shen</strong> , 2016
                <p>
                  <br> A English-Chinese Quiz auto-maker with GUI to help Chinese students learn difficult English words for Graduate Record Examinations(GRE) which can prepare both the quiz and the answers.
                  </br>
                  <br> <a href="https://github.com/iphyer/Crack3kGREWords">
                  <papertitle>Github(In Chinese)</papertitle></br></a>
                </p>
              </p>
            </td>
          </tr>
        </table> -->


<!--         
=====================================================================
                        Footer 
==================================================================== 
-->

        <script type="text/javascript">
          var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
          document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
        </script>
        <script type="text/javascript">
          try {
            var pageTracker = _gat._getTracker("UA-133310834-1");
            pageTracker._trackPageview();
          } catch (err) {}
        </script>
        </td>
    </tr>
  </table>
</body>

</html>
